project/MicroService.scala

```diff
@@ -27,7 +27,23 @@ trait MicroService {
   lazy val microservice = Project(appName, file("."))
     .enablePlugins(Seq(play.sbt.PlayScala,SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin) ++ plugins : _*)
     .settings(playSettings : _*)
-    .settings(scalaSettings: _*)
+    .settings( Seq(
```

Comment:
Can we change this back again to scalasettings since we only needed the upgraded version for scalafix

Reply:
changed